design-tokens/tokens/layer2/dark.json

```diff
@@ -1,76 +1,316 @@
 {
-
-  "boxShadow": {
-    "default": {
-      "value": [
-        {
-          "x": 5,
-          "y": 5,
-          "spread": 3,
-          "color": "rgba({color.shadows.default}, 0.15)",
-          "blur": 5,
-          "type": "dropShadow"
-        },
-        {
-          "x": 4,
-          "y": 4,
-          "spread": 6,
-          "color": "#00000033",
-          "blur": 5,
-          "type": "innerShadow"
+  "echoes": {
+    "color": {
+      "background": {
+        "neutral": {
+          "type": "color",
+          "value": "{color.grey.500}"
+        },
+        "neutral-bolder": {
+          "type": "color",
+          "value": "{color.grey.400}"
+        },
+        "default": {
+          "type": "color",
+          "value": "{color.grey.600}"
+        },
+        "default-hover": {
+          "type": "color",
+          "value": "{color.grey.700}"
+        },
+        "default-active": {
+          "type": "color",
+          "value": "{color.grey.700}"
+        },
+        "default-disabled": {
+          "type": "color",
+          "value": "{color.grey.500}"
+        },
+        "page": {
+          "type": "color",
+          "value": "{color.grey.700}"
+        },
+        "emphasize": {
+          "type": "color",
+          "value": "{color.grey.700}"
+        },
+        "inverse": {
+          "type": "color",
+          "value": "{color.grey.100}"
+        },
+        "disabled": {
+          "type": "color",
+          "value": "{color.grey.500}"
+        },
+        "accent": {
+          "default": {
+            "accent": {
+              "type": "color",
+              "value": "{color.indigo.200}"
+            },
+            "accent-hover": {
+              "type": "color",
+              "value": "{color.indigo.300}"
+            },
```

Comment:
variable names are not gonna make any sense:
`--echoes-color-background-accent-default-accent`

We need to remove the first accent object or remove the second `accent` in the name.

By the way, this is the same for all the following design tokens like weak-weak, warning-default-warning, success-weak-weak, danger-..., info-...